homeassistant/components/onewire/sensor.py

```diff
@@ -120,9 +131,25 @@ def hb_info_from_type(dev_type="std"):
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up 1-Wire platform."""
-    base_dir = config[CONF_MOUNT_DIR]
-    owport = config[CONF_PORT]
+    devs = get_entities(config)
+    add_entities(devs, True)
+
+
+def get_entities(config):
+    """Get a list of entities."""
+    conf_type = config.get(CONF_TYPE)
```

Comment:
`CONF_TYPE` is not in the platform config schema. We don't allow changes to platform config anymore. If we need to change the config structure, this integration needs to be migrated to have the config under the integration domain key.